device/common/include/traccc/clusterization/device/impl/ccl_kernel.ipp

```diff
@@ -260,10 +262,10 @@ TRACCC_DEVICE inline void ccl_kernel(
          * any).
          */
         while (start != 0 && start < num_cells &&
-               cells_device.at(start - 1).module_link ==
-                   cells_device.at(start).module_link &&
-               cells_device.at(start).channel1 <=
-                   cells_device.at(start - 1).channel1 + 1) {
+               cells_device.module_index().at(start - 1) ==
+                   cells_device.module_index().at(start) &&
```

Comment:
In this file: please use proxy syntax.

Reply:
However in this file I didn't see much benefit to using a proxy syntax, even now. :thinking: